examples/python/mast3r_slam/README.md

```diff
@@ -0,0 +1,24 @@
+<!--[metadata]
+title = "Mast3r SLAM - Real-Time Dense SLAM with 3D Reconstruction Priors"
+tags = ["2D", "3D", "Pinhole camera", "Time series", "SLAM"]
+source = "https://github.com/rerun-io/mast3r-slam"
+thumbnail = "https://static.rerun.io/thumbnail/3659cc28fb5ab6173f930e26dd8158f1638eb284/480w.png"
+thumbnail_dimensions = [480, 301]
```

Comment:
Square aspect ratio works _much_ better on our landing page. Could you adjust the thumbnail for that?